src/duct/core/env.clj

```diff
@@ -12,6 +13,14 @@
 (defmethod coerce 'Str [x _]
   (str x))
 
+(defmethod coerce 'Bool [x _]
+  (let [x-lower-case (some-> x str/lower-case)]
+    (cond
+      (contains? #{"true" "t" "y" "yes"} x-lower-case) true
+      (contains? #{"false" "f" "n" "no" nil ""} x-lower-case) false
+      :else (throw (IllegalArgumentException.
```

Comment:
Rather than `IllegalArgumentException`, perhaps using `ex-info` would be more Clojurey. Something like:

```clojure
(throw (ex-info (str "Could not coerce " (pr-str x) " into a boolean."
                     "Must be one of: \"true\", \"t\", \"false\", \"f\","
                     "\"yes\", \"y\", \"no\", \"n\", \"\" or nil.")
                {:value x, :coercion 'Bool}))
```

Also, can you keep lines to under 80 characters?

Reply:
Was using IllegalArgumentException to follow the 'Int coercion. Changed to ex-info